src/plugins/intel_cpu/src/nodes/reorder.cpp

```diff
@@ -471,6 +471,34 @@ std::vector<VectorDims> Reorder::shapeInfer() const {
     return {getParentEdgesAtPort(0)[0]->getMemory().getStaticDims()};
 }
 
+void Reorder::reorderDataUsingCache(MultiCachePtr cache, const Memory &input, const Memory &output) {
```

Comment:
We definitely should merge both the two `Reorder::reorderData` and `Reorder::reorderDataUsingCache` calls. The new symbol should be as follows:
```cpp
   void Reorder::reorderData(const Memory &input, const Memory &output, MultiCachePtr cache = nullptr)
```
That means default call `reorderData(input, output)` leads to a temporal instance of the reorder primitive. When user provides cache, cache lookup happens.

Reply:
Merged.